RELEASE-NOTES.md

```diff
@@ -0,0 +1,15 @@
+# v2.0.0
+
+## Backward incompatibility
+* In `snowpark function` command:
+  * Combined options `--function` and `--input-parameters` to `identifier` argument
+  * Changed name of option from `--return-type` to `return`
```

Comment:
In the DDL it is `returns`, should we align with that?

Reply:
Good idea 👍